packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -53,6 +56,7 @@ export enum ProjectionName {
   Address = 'address',
   Asset = 'asset',
   Handle = 'handle',
+  Parameters = 'parameters',
```

Comment:
```suggestion
  ProtocolParameters = 'protocol-parameters',
```

Reply:
Let's keep it consistent. Protocol Parameters is more identifiable.

Reply:
ded3e97e6d1 should solve this